src/App.js

```diff
@@ -42,10 +40,33 @@ class App extends React.Component {
     this.setState({ show: false });
   };
 
+  handleChange = (event) => {
+    event.preventDefault();
+    const selection = parseInt(event.target.value);
+    let hornAmount;
+    if (selection === 1) {
```

Comment:
So you're repeating yourself a bit here in your code. When you declare hornAmount you can do your filtering there once.
```js
const selection = parseInt(event.target.value); // note: I made your value for "All Bests" 0 to make this code work. That way everything is a number.
const let hornAmount = json.filter(amount => amount.horns === selection);
```
Now, here you can either do an if else, or a ternary.
```js
selection > 0 ? this.setState({data: hornAmount}) : this.setState({data: json});
```